app/api/endpoints/lossClaim/list/v3/ListLossClaimsService.scala

```diff
@@ -39,15 +39,28 @@ class ListLossClaimsService @Inject()(connector: LossClaimConnector) extends Dow
     }
   }
 
-  private def errorMap: Map[String, MtdError] = Map(
-    "INVALID_TAXABLE_ENTITY_ID" -> NinoFormatError,
-    "INVALID_TAXYEAR"           -> TaxYearFormatError,
-    "INVALID_INCOMESOURCEID"    -> BusinessIdFormatError,
-    "INVALID_INCOMESOURCETYPE"  -> TypeOfLossFormatError,
-    "INVALID_CLAIM_TYPE"        -> TypeOfClaimFormatError,
-    "NOT_FOUND"                 -> NotFoundError,
-    "INVALID_CORRELATIONID"     -> InternalError,
-    "SERVER_ERROR"              -> InternalError,
-    "SERVICE_UNAVAILABLE"       -> InternalError
-  )
+  private def errorMap: Map[String, MtdError] = {
+    val downstreamErrors = Map(
```

Comment:
Might be little bit better to have e.g. `errors` and `extraTysErrors` here instead, as tys errors are downstream as well.